### ISSUE TITLE
Fan dimmer not working in alexa App

### ISSUE THREAD

**Issue by sonetonix-iot** [espressif/esp-rainmaker#142]
All parameters are listed in alexa App. i can control lights/switches from the alexa App,but i can't able to control dimmer/fan speed using alexa app.

**Comment by jamsyogendra** [comment]
hello i am working on same project can you halp me

**Comment by jamsyogendra** [comment]
https://github.com/espressif/esp-rainmaker/issues/139#issue-1311268564

**Comment by sonetonix-iot** [comment]
> hello i am working on same project can you halp me

how you integrate the IR Remote control with this esp rain maker App

**Comment by jamsyogendra** [comment]
using IR lib

**Comment by jamsyogendra** [comment]
#include <IRremote.h>

**Comment by sonetonix-iot** [comment]
did you control IR function from ESP RAINMAKER App ?

**Comment by jamsyogendra** [comment]
yes i did it also with manually feadback with manual switch and ir switch

**Comment by sonetonix-iot** [comment]
it means if you press any button in App, the user's custom IR command will illuminate right?

**Comment by jamsyogendra** [comment]
yes

**Comment by sonetonix-iot** [comment]
Okay thank you

**Comment by jamsyogendra** [comment]
whats about fan dimmer which technics are you using from control fan speed or control brigtness of AC. 

Zerocross of relays base?

**Comment by sonetonix-iot** [comment]
both techs used ...circuit side no problem

**Comment by jamsyogendra** [comment]
can you share some details about zero cross dimming method

**Comment by sonetonix-iot** [comment]
what type of bulb led or incandecent

**Comment by jamsyogendra** [comment]
ac bulb or we can say ac dimmer 

https://www.tinytronics.nl/shop/en/robotdyn-thyristor-ac-dimmer-3.3-5v

**Comment by jamsyogendra** [comment]
Anything solution

**Comment by sonetonix-iot** [comment]
there will be an example program, as you mentioned link. it will work

**Comment by CC-MASTER05** [comment]
@jamsyogendra are you able to dimming your ac fan if yes can you please share your time sync function with zero-cross detection?
